ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -458,11 +458,28 @@ def test_ipa_trust_check_exists(self):
         Testcase to verify checks available in
         ipahealthcheck.ipa.trust source
         """
+        # earlier trust check used to return an empty SUCCESS message when
+        # trust is not configured.But now it actually returns a message
+        not_a_trust_agent = ["IPATrustAgentCheck", "IPATrustCatalogCheck",
+                             "IPAsidgenpluginCheck", "IPATrustAgentMemberCheck",
+                             "IPATrustDomainsCheck", "IPATrustPackageCheck"]
+
+        not_a_trust_controller = ["IPATrustControllerPrincipalCheck",
+                                  "IPATrustControllerServiceCheck",
+                                  "IPATrustControllerConfCheck",
+                                  "IPATrustControllerGroupSIDCheck",
+                                  "IPATrustControllerAdminSIDCheck"]
```

Comment:
you may want to add this check (IPATrustControllerAdminSIDCheck) in the list ipatrust_checks (currently we have:
```
ipatrust_checks = [
    "IPATrustAgentCheck",
    "IPATrustDomainsCheck",
    "IPADomainCheck",
    "IPATrustCatalogCheck",
    "IPAsidgenpluginCheck",
    "IPATrustAgentMemberCheck",
    "IPATrustControllerPrincipalCheck",
    "IPATrustControllerServiceCheck",
    "IPATrustControllerConfCheck",
    "IPATrustControllerGroupSIDCheck",
    "IPATrustPackageCheck",
]
```

Reply:
Sure will add it.